lib/App.js

```diff
@@ -1833,20 +1826,164 @@ class App {
             return true;
           }
         }
-      ]);
+      ]); 
 
-      // Create a custom object to inject en flag for the placeholder.
-      flowArguments.push({
-        type: argumentType,
-        name: argumentName,
-        placeholder: { en: argumentPlaceholder },
-      });
+      if (argumentType != "dropdown" && argumentType != "color" && argumentType != "droptoken" && argumentType != "range"){
+        const { argumentPlaceholder } = await inquirer.prompt([
+          {
+            type: 'input',
+            name: 'argumentPlaceholder',
+            message: 'Enter the placeholder for the argument',
+            validate: input => input.length > 0,
+          }
+        ]);
+        if(argumentType == "number"){
+          const { argumentMin, argumentMax, argumentStep } = await inquirer.prompt([
+            {
+              type: 'input',
+              name: 'argumentMin',
+              message: 'Enter the minimum value for the argument',
+              default: 0,
+              validate: input => !Number.isNaN(Number(input))
+            },
+            {
+              type: 'input',
+              name: 'argumentMax',
+              message: 'Enter the maximum value for the argument',
+              default: 100,
+              validate: input => !Number.isNaN(Number(input))
+            },
+            {
+              type: 'input',
+              name: 'argumentStep',
+              message: 'Enter the step for the argument',
+              default: 1,
+              validate: input => !Number.isNaN(Number(input))
+            },
+          ]);
+          // Create a custom object without a placeholder.
+        flowArguments.push({
+          type: argumentType,
+          name: argumentName,
+          placeholder: { en: argumentPlaceholder },
+          min: argumentMin,
+          max: argumentMax,
+          step: argumentStep
+        }); 
+
+        } else{
```

Comment:
It's better to do something like `flowArgument.placeholder = { en: argumentPlaceholder }`. Now you have a set of types that can never have a placeholder. So keep your code together.